database/migrations/2023_09_25_044810_update_landing_pages_tables_from_pages_table.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        DB::statement("INSERT INTO landing_page_types (page_type) VALUES ('Pre-made Tours');");
+    }
```

Comment:
page_type is set to Pre-Made Tours to avoid templating clash with page_type = custom